gobblin-core-base/src/main/java/org/apache/gobblin/converter/filter/AvroProjectionConverter.java

```diff
@@ -74,10 +74,15 @@ public AvroProjectionConverter init(WorkUnitState workUnit) {
    */
   @Override
   public Schema convertSchema(Schema inputSchema, WorkUnitState workUnit) throws SchemaConversionException {
+    Schema outputSchema = inputSchema;
     if (this.fieldRemover.isPresent()) {
-      return this.fieldRemover.get().removeFields(inputSchema);
+      outputSchema = this.fieldRemover.get().removeFields(inputSchema);
     }
-    return inputSchema;
+    if(workUnit.getPropAsBoolean(ConfigurationKeys.CONVERTER_AVRO_INCLUDE_SCHEMA_CREATION_TIME,
+        ConfigurationKeys.DEFAULT_CONVERTER_AVRO_INCLUDE_SCHEMA_CREATION_TIME)) {
```

Comment:
what if a pipeline needs this but not using this converter?  shall we consider to add it into higher level ?

Reply:
my concern is that will make the change not backward compatible, and most converter will not erase the schema prop, how about your thoughts?

Reply:
Given the timestamp is set in the properties of an Avro schema object, does that matter for other writers that are not aware of the existence of this property ? We can try with a simple unit test to see if setting properties will impact writer. 

I mostly concern on the operability problem since there's no guarantee that this converter is and will used everywhere.